### ISSUE TITLE
Batch Normalization Layer

### ISSUE THREAD

**Issue by Hitomamacs** [ZIGTinyBook/Z-Ant#122]
### Feature Description

Add the Batch Normalization layer, it must be compatible with tabular and images data, and correct the bug of normalization in model for the moment do not consider learnable parameter gamma and beta for mean and std dev. just consider 0 and 1

### Problem Statement

Right now in model forward there is something similar to batch norm, it needs to be eliminated and transformed in the batch norm layer. More over look at the normalization process, it seems that the nn behave in a strange way without batchnorm, give  look at it.

### Suggested Solution

New layer as the one before, and testing of the network with different structures a problem might be the shitty optimizer

### Additional Context

Add any other context, designs, or screenshots related to the request.

<sub><a href="https://huly.app/guest/thetinybook?token=eyJ0eXAiOiJKV1QiLCJhbGciOiJIUzI1NiJ9.eyJsaW5rSWQiOiI2Nzc3YjRlM2YxNDRmNTg1YWU1MjRlZDAiLCJndWVzdCI6InRydWUiLCJlbWFpbCI6IiNndWVzdEBoYy5lbmdpbmVlcmluZyIsIndvcmtzcGFjZSI6InctZ2l0aHViZmxhbWFuLXRoZXRpbnlib29rLTY3MGU4NzBiLWRjYzkyNjFmMDItMTMxM2JkIn0.3fiCZA4y8W7EyYx2gy9anq8Ilk5uf-mmuS92WZ9s47s">Huly&reg;: <b>X_PI-154</b></a></sub>

**Comment by Hitomamacs** [comment]
New better issue

**Comment by Hitomamacs** [comment]
f
